packages/react-card/src/components/Card/Card.types.ts

```diff
@@ -6,6 +6,7 @@ export type CardSlots = {
 
 export type CardCommons = {
   appearance: 'filled' | 'filled-alternative' | 'outline' | 'subtle';
+  focusable: boolean | 'noTab' | 'tabExit' | 'tabOnly';
```

Comment:
I agree on the string values point. With that change `focusable` doesn't feel like the correct name for this prop anymore. What do you think about `focusMode`?

Reply:
**question/thought:** is there a generic enough pattern we could extract and apply around all/most of our controls regarding focusable behaviours and APIs ?

Reply:
@Hotell I think it is. There is a similar need for the `AvatarGroup` component that @sopranopillow is building.

Reply:
My preference is for there to be a single prop that controls this behavior because, as you noted, you get into situations where `focusMode` does nothing which can be surprising for users. What do you think about just having:

`focusMode: 'off' | 'no-tab' | 'tab-exit' | 'tab-only'`

The component then applies `tabIndex` and Tabster attrs based solely on the `focusMode` value which eliminates (or at least minimizes) the possibility of having contradictory values being applied.

Reply:
I like that approach! I'll go ahead and implement it should there be no further discussion.